scripts/travis/.travis.yaml

```diff
@@ -0,0 +1,59 @@
+language: android
```

Comment:
We can remove travis config and all the travis script files before we merge this PR.